app/models/spree/shipment_decorator.rb

```diff
@@ -1,27 +1,35 @@
-Spree::Shipment.class_eval do
-  def avatax_cache_key
-    key = ['Spree::Shipment']
-    key << id
-    key << cost
-    key << stock_location.cache_key
-    key << promo_total
-    key.join('-')
-  end
+# frozen_string_literal: true
 
-  def avatax_line_code
-    'FR'
-  end
+module Spree
+  # ShipmentDecorator
```

Comment:
I think it makes no sense to copy the name of the class/module as documentation...